cmd/codegen/generator/go/generator.go

```diff
@@ -385,23 +389,28 @@ package main
 
 import (
 	"context"
+	"%[2]s/internal/dagger"
```

Comment:
I suppose now if I never run `dagger develop` and the generated module code isn't committed, instead of my editor giving me a bunch of errors for undefined types, there will just be this one unknown import, which is better

Reply:
That too I guess! The main reason for this in here is that if you *don't* do it, go cleverly attempts to "find it" - and often seems to select `dagger.io/dagger` which is reasonably unhelpful.

Reply:
I think that issue still exists but for `dag` right? I remember that auto-importing from somewhere. :thinking: Or is that fixed by now too?

Reply:
Ah, that issue exists for users if they don't run `dagger develop` first - since it's in `dagger.gen.go`. Sadly, I don't know about how to fix that one :thinking: